### DISCUSSION TITLE
Add a "ready check" function to the app?

### DISCUSSION THREAD

**Discussion by sweetroll482** [OhMyGuus/BetterCrewLink#337] [Ideas]
Just a checkbox next to each name. That's all it would take.

**Comment by MatadorProBr** [comment]
A ready check for those who has BCL opened or something? Explain more about this "ready check" and how it would work

**Reply by sweetroll482** [reply]
Just an idea for the BCL app in the pre-game lobby.

Either make each crewmate's picture clickable to show a checkmark, or add a checkbox for everyone. That way the lobby leader will know for sure when everyone is ready to begin the next game.

**Comment by sweetroll482** [comment]
If you were to just add a checkbox function to everyone (maybe make it
where we can click our crewmate pictures in the BCL app) the lobby leader
could better be able to see when everyone is ready to start the next game.
